CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ We have several packages which live in this repository. Changes are tracked sepa
 [defmt-v0.1.0]: https://github.com/knurling-rs/defmt/releases/tag/defmt-v0.1.0
 
 ### [defmt-next]
+* [#894] Add optional support for bitflags v2
```

Comment:
There is another change here already in `main`. You'll need to rebase.

Reply:
done

Reply:
sorry for the force pushes :)